modules/pytket-quantinuum/setup.py

```diff
@@ -38,7 +38,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.2",
+        "pytket == 1.3.0rc1",
```

Comment:
Should we change the current version of the extension of the extension to RC, too?